src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/patches/PatchListSearchValue.java

```diff
@@ -15,13 +17,22 @@ public String getAuthor() {
         return author;
     }
 
+    public String getTag() {
+        return tag;
+    }
+    public String getState() {
+        return state;
+    }
+
     public PatchListSearchValue() {
```

Comment:
All constructors should be the first methods in their class.
Why isn't this checked by checkstyle already? 
I will create an issue for this.